openshift-hack/commitchecker/commitchecker.go

```diff
@@ -29,6 +29,8 @@ func main() {
 		}
 	}
 
+	errs = append(errs, ValidateReleaseBranchConsistency(commits)...)
```

Comment:
Instead of doing it here, add `ValidateReleaseBranchConsistency` in openshift-hack/commitchecker/validate.go in `AllCommitValidators`

Reply:
yeah I tried, I didn't want to change the existing function on a single commit though. This needs to first check and fetch the remote release branches and then for the respective commit in a branch. 

It's doing different things :/ Shall we change the `func(Commit)` in `AllCommitValidators` to take a branch as well?

Reply:
moved the whole thing to output to stdout first instead of returning an err